pkg/ddevapp/config.go

```diff
@@ -90,6 +106,16 @@ func (c *Config) Write() error {
 		return err
 	}
 
+	cfgbytes = append(cfgbytes, []byte(HookTemplate)...)
+	switch c.AppType {
+	case "drupal8":
+		cfgbytes = append(cfgbytes, []byte(Drupal8Hooks)...)
+	case "drupal7":
+		cfgbytes = append(cfgbytes, []byte(Drupal7Hooks)...)
+	case "wordpress":
+		cfgbytes = append(cfgbytes, []byte(WordPressHooks)...)
+	}
+
```

Comment:
to be clear, I'm also not advocating for changes here. Just spitballing about how we may make it more sustainable going forward.